specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2023-07-12-preview/AlertsManagement.json

```diff
@@ -0,0 +1,1568 @@
+          "items": {
+            "$ref": "#/definitions/alertsSummaryGroupItem"
+          },
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "alertsSummaryGroupItem": {
+      "type": "object",
+      "description": "Alerts summary group item",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Value of the aggregated field"
+        },
+        "count": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Count of the aggregated field"
+        },
+        "groupedby": {
+          "type": "string",
+          "description": "Name of the field aggregated"
+        },
+        "values": {
+          "type": "array",
+          "description": "List of the items",
+          "items": {
+            "$ref": "#/definitions/alertsSummaryGroupItem"
+          },
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "alertsMetaData": {
+      "type": "object",
+      "description": "alert meta data information.",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/alertsMetaDataProperties"
+        }
+      }
+    },
+    "alertsMetaDataProperties": {
+      "type": "object",
+      "description": "alert meta data property bag",
+      "discriminator": "metadataIdentifier",
+      "required": [
+        "metadataIdentifier"
+      ],
+      "properties": {
+        "metadataIdentifier": {
+          "type": "string",
+          "description": "Identification of the information to be retrieved by API call",
+          "enum": [
+            "MonitorServiceList"
+          ],
+          "x-ms-enum": {
+            "name": "metadataIdentifier",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "MonitorServiceList": {
+      "type": "object",
+      "title": "Monitor service details",
+      "description": "Monitor service details",
+      "required": [
+        "data"
+      ],
+      "allOf": [
+        {
+          "$ref": "#/definitions/alertsMetaDataProperties"
+        },
+        {
+          "type": "object"
+        }
+      ],
+      "properties": {
+        "data": {
+          "description": "Array of operations",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/MonitorServiceDetails"
+          },
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "MonitorServiceDetails": {
+      "type": "object",
+      "description": "Details of a monitor service",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Monitor service name"
+        },
+        "displayName": {
+          "type": "string",
+          "description": "Monitor service display name"
+        }
+      }
+    },
+    "alertEnrichmentResponse": {
+      "description": "The alert's enrichments.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
```

Comment:
>Resource [](http://example.com/codeflow?start=33&length=8)

Recommend getting this from v5 common types.

Reply:
done.